### ISSUE TITLE
Dragging blocks into "text" (as oppossed to drop-areas)

### ISSUE THREAD

**Issue by braders** [WordPress/gutenberg#6238]
## Issue Overview
It is possible to drop a block into a text area, which inserts the internal representation of the block being moved.

## Steps to Reproduce (for bugs)
1. Start dragging a block
2. Carefully position the cursor at the initial carat of a text area.
3. Drop the block / release the mouse click

I am using Firefox Developer Edition (Firefox 60); Windows 10.

It is also possible to drag a block into the browser address bar and get the same internal representation - this seems like much less of a problem though (and may not be resolvable). 

## Screenshots / Video
![guttenburg-drag-into-text-bug](https://user-images.githubusercontent.com/1931323/38922786-ea5d29b2-42f0-11e8-869d-1e3c2ede4015.gif)

## Todos
- [ ] Tests
- [ ] Documentation

**Comment by AtrumGeost** [comment]
Hi there,

I tested this with Gutenberg 4.0 and wasn't able to reproduce the issue. It worked properly on my end:

![test gh 6238](https://user-images.githubusercontent.com/3696121/47182663-a0b27e80-d2eb-11e8-84f1-85745334a36a.gif)

As for moving the block to the browser address bar: Yes, I was able to reproduce that. But doesn't seem to be a problem that could have a negative effect.

_WordPress 4.9.8 and Gutenberg 4.0.0 using Google Chrome Version 69.0.3497.100 (Official Build) (64-bit) on macOS 10.14 - Mac OS Mojave_
